examples/blog-with-comment/.env.local.example

```diff
@@ -1,4 +1,14 @@
-REDIS_URL=
-NEXT_PUBLIC_AUTH0_CLIENT_ID=
-NEXT_PUBLIC_AUTH0_DOMAIN=
-NEXT_PUBLIC_AUTH0_ADMIN_EMAIL=
+UPSTASH_REDIS_REST_URL=
+UPSTASH_REDIS_REST_TOKEN=
+
+# https://github.com/settings/apps
+GITHUB_ID=
+GITHUB_SECRET=
+NEXT_PUBLIC_ADMIN_EMAIL=
+
+# Secret key
+# generate one here: https://generate-secret.vercel.app/32
+NEXTAUTH_SECRET=
+
+# Mandatory next-auth URL for localhost
+NEXTAUTH_URL=http://localhost:3000
```

Comment:
```suggestion
```

It's not, it's the default value. It shows up as a warning, but it is a bug in `next-auth`. Will try to address here: https://github.com/nextauthjs/next-auth/pull/5763